include/shisen_cpp/config/grpc/call_data.hpp

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) 2024 Ichiro ITS
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+// THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+#ifndef __SHISEN_CPP__CONFIG__GRPC__CALL_DATA_HPP__
+#define __SHISEN_CPP__CONFIG__GRPC__CALL_DATA_HPP__
+
+#include <grpc/support/log.h>
+#include <grpcpp/grpcpp.h>
+#include <shisen_cpp/config/grpc/call_data_base.hpp>
+#include <shisen_interfaces/shisen.grpc.pb.h>
+#include <shisen_interfaces/shisen.pb.h>
+
+enum class CallStatus
+{
+  CREATE  = 1,
+  PROCESS = 2,
+  FINISH  = 3,
+};
```

Comment:
No need to specify the value of each entry in an enum class:
```suggestion
enum class CallStatus
{
  CREATE,
  PROCESS,
  FINISH,
};
```